ui/apps/platform/src/services/CloudSourceService.ts

```diff
@@ -26,6 +26,11 @@ export type CloudSourceIntegration = {
     ocm?: OcmConfig;
 };
 
+export type UpdateCloudSourceRequest = {
```

Comment:
An observation, not a change request:
* Pro that convinces me: exact same name as backend source of truth
    https://github.com/stackrox/stackrox/blob/master/proto/api/v1/cloud_source_service.proto#L102-L108
* Con that I learn to read: `request` argument below is a part of the request:
    https://developer.mozilla.org/en-US/docs/Web/API/Request/Request